<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>designCalDialog</class>
 <widget class="QWidget" name="designCalDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>241</width>
    <height>87</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>236</width>
     <height>61</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="radiusLabel">
        <property name="text">
         <string>&amp;半径：</string>
        </property>
        <property name="buddy">
         <cstring>radiusLineEdit</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="radiusLineEdit"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="areaLabel_1">
        <property name="text">
         <string>面积：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="areaLabel_2">
        <property name="frameShape">
         <enum>QFrame::Panel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Sunken</enum>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="countBtn">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>计算</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>radiusLineEdit</tabstop>
  <tabstop>countBtn</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
